crypto/sleep.c

```diff
@@ -46,27 +46,15 @@ void OSSL_sleep(uint64_t millis)
 {
     /*
      * Windows' Sleep() takes a DWORD argument, which is smaller than
-     * a uint64_t, so we need to split the two to shut the compiler up.
+     * a uint64_t, so we need to limit it to 49 days, which should be enough.
      */
-    DWORD dword_times;
-    DWORD i;
+    DWORD milliseconds = (DWORD)-1;
```

Comment:
at this point the variable is really `max_millis`, but then becomes `actual_millis`. Since we get a cast either way, why not just do

```
DWORD max_millis = (DWORD)-1;

if (millis > max_millis) {
    millis = max_millis;
}

Sleep((DWORD)millis);
```

The same applies below a couple of times, too

Reply:
I prefer the current way.

Reply:
Okay, let me try it another way: there are two variables that each hold a number of milliseconds. The name of one is abbreviated, but not the name of the other. Neither has anything in the name that indicates which was the input parameter, which is the maximum amount of time we will sleep for, and which is the value that will be used for the actual time to sleep.